railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -37,3 +37,10 @@
 
 # Set both the `:open_timeout` and `:read_timeout` values for `:smtp` delivery method.
 # Rails.application.config.action_mailer.smtp_timeout = 5
+
+# Uses a specific purpose signed id for direct uploads instead of using the same signed id
+# used in other places. Changing this default means that ActiveStorage won't continue to attach
+# blobs by their old signed id `purpose: :blob_id`. It is recommended to first deploy Rails 7
+# with the previous defaults and then uncomment the flag to be `false`.
```

Comment:
```
# Sign direct uploads with a unique purpose.
#
# Direct uploads should now signed with a :direct_upload purpose. Previously, they were signed with :blob_id. 
# If you use direct uploads, your app will need to temporarily be able to find both signed types.
# Make sure your entire app is migrated and stable on 7.0 before using this setting.
#
# Rails.application.config.active_storage.direct_upload_purpose_signing = false
```

Somewhat similar to https://github.com/rails/rails/pull/34692/files#diff-a10710b71e54cf750d72df8516aa9ca7a01a4239bdc9b36575208f116cae406a. I think something like this would sound better.